lib/runtime/life/instance.go

```diff
@@ -41,9 +41,8 @@ type Config struct {
 
 // Instance represents a v0.8 runtime life instance
```

Comment:
Should we amend this comment? :thinking: Since the version isn't contained in the instance and also it wasn't fixed :thinking:

Reply:
Indeed, this should probably be updated.